poc/vdaf_pine.py

```diff
@@ -105,26 +108,39 @@ def __init__(self,
                  l2_norm_bound: int,
                  num_frac_bits: int,
                  dimension: int,
-                 chunk_length: int,
+                 chunk_length_squared_norm_equality: int,
```

Comment:
The optimal `chunk_length` can be quite different between `PineValidSquaredNormEqualityCheck` and `PineValidOtherChecks`. It makes senses to introduce a new parameter?